### ISSUE TITLE
Command not found when using action-apt

### ISSUE THREAD

**Issue by bdelamotte** [ryankurte/action-apt#5]
```
 runs-on: ubuntu-latest
...
  - uses: ryankurte/action-apt@v0.1.0
        with:
          packages: |
            libsasl2-dev
```
I get an error 

> 
> Run ryankurte/action-apt@v0.1.0
>   with:
>     packages: libsasl2-dev
>   
> /usr/bin/sudo ./patch-apt.sh /etc/apt/
> sudo: ./patch-apt.sh: command not found
> Error: The process '/usr/bin/sudo' failed with exit code 1

**Comment by ryankurte** [comment]
hey thanks for the report! v weird, it _shouldn't_ try to patch that if there's no architecture specified. just pushed an update with another check [here](https://github.com/ryankurte/action-apt/blob/master/src/main.ts#L11) which i think you could try from `ryankurte/action-apt@master`?

(as an aside, you can always `run: sudo apt update && sudo apt install ...` if you need)

**Comment by bdelamotte** [comment]
Thanks for looking at this! I got around the issue by doing a normal run with sudo apt-get commands. Today I will try again with `ryankurte/action-apt@master` as you've suggested.

**Comment by ryankurte** [comment]
yeah the sudo commands solve most of the problem, i only really added this to get multiarch working more consistently
